.env.example

```
@@ -0,0 +1,15 @@
+# Copy this file to .env and fill in values.
+
+# Azure Blob Storage — required for CGM data (stage 6)
+AZURE_ACCOUNT_NAME=your_storage_account_name
+AZURE_SAS_TOKEN=your_sas_token
+AZURE_CONTAINER_NAME=your_container_name
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AZURE_CONTAINER_NAME key should go before the AZURE_SAS_TOKEN key

(UnorderedKey)